src/OnboardingSPA/steps/SiteGen/SocialMedia/index.js

```diff
@@ -22,12 +24,40 @@ const SiteGenSiteSocialMedia = () => {
 		setHeaderActiveView( HEADER_SITEGEN );
 		setDrawerActiveView( false );
 	} );
+
+	const content = getContents();
 	return (
 		<CommonLayout
 			isCentered
 			className="nfd-onboarding-step--site-gen__social-media"
 		>
-			<SiteGenPlaceholder heading={ 'Social Media' } />
+			<div className="nfd-onboarding-step--site-gen__social-media__container">
+				<div className="nfd-onboarding-step--site-gen__social-media__container__heading">
+					<div className="nfd-onboarding-step--site-gen__social-media__container__heading__animation"></div>
+					<p className="nfd-onboarding-step--site-gen__social-media__container__heading__text">
+						{ content.heading }
+					</p>
+				</div>
+				<div className="nfd-onboarding-step--site-gen__social-media__contain ">
+					<div className="nfd-onboarding-step--site-gen__social-media__contain__containleft ">
+						{ content.facebookTitle }
+						<p>{ content.facebookdesc }</p>
+					</div>
+					<div className="nfd-onboarding-step--site-gen__social-media__contain__containright ">
+						<button className="nfd-onboarding-step--site-gen__social-media__contain__containright__button ">
```

Comment:
I think the icon is supposed to be white? 
<img width="1617" alt="Screenshot 2023-11-20 at 1 18 49 PM" src="https://github.com/newfold-labs/wp-module-onboarding/assets/48691514/5c033abf-42aa-46b4-b437-b18dd26961cf">